source/github-helpers/get-default-branch.ts

```diff
@@ -59,7 +53,13 @@ const getDefaultBranch = cache.function(async function (repository?: pageDetect.
 }, {
 	maxAge: {hours: 1},
 	staleWhileRevalidate: {days: 20},
-	cacheKey: ([repository = getRepo()]) => 'default-branch:' + repository!.nameWithOwner,
+	cacheKey: ([repository]) => 'default-branch:' + repository.nameWithOwner,
 });
 
-export default getDefaultBranch;
+export default async function getDefaultBranch(repository: pageDetect.RepositoryInfo | undefined = getRepo()): Promise<string> {
+	if (!repository) {
+		throw new Error('getDefaultBranch was called on a non-repository page');
+	}
+
+	return _getDefaultBranch(repository);
+}
```

Comment:
The changes to this file can be reverted once webext-storage-cache is fixed:

- https://github.com/fregante/webext-storage-cache/issues/34